src/span_context.js

```diff
@@ -98,6 +98,20 @@ export default class SpanContext {
     return this._spanIdStr;
   }
 
+  get toTraceId(): ?string {
+    if (this._traceIdStr == null && this._traceId != null) {
```

Comment:
why not simply `return this.traceIdStr`?

Reply:
Tried it. But flow kept giving me an error which I couldn't figure out 
```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/span_context.js:109:17

Cannot call this.spanIdStr because:
 • null or undefined [1] is not a function.
 • string [2] is not a function.

 [1][2]  94│   get spanIdStr(): ?string {
           :
        106│   }
        107│
        108│   get toSpanId(): ?string {
        109│     return this.spanIdStr()
        110│     // if (this._spanIdStr == null && this._spanId != null) {
        111│     //   this._spanIdStr = Utils.removeLeadingZeros(this._spanId.toString('hex'));
        112│     // }
```

Not really familiar with flow tbh

Reply:
`get` means it's a property, but you're invoking it as a function.

Reply:
Yup! My mistake. This is the first time I'm actually playing around with JS getters and setters